employee-rostering-benchmark/src/main/resources/employeeRosteringBenchmarkConfig.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plannerBenchmark>
+  <benchmarkDirectory>local/benchmarkReport</benchmarkDirectory>
+  <warmUpSecondsSpentLimit>30</warmUpSecondsSpentLimit>
+
+  <inheritedSolverBenchmark>
+    <solver>
+      <solutionClass>org.optaweb.employeerostering.domain.roster.Roster</solutionClass>
+      <entityClass>org.optaweb.employeerostering.domain.shift.Shift</entityClass>
+
+      <scoreDirectorFactory>
+        <scoreDrl>org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl</scoreDrl>
+      </scoreDirectorFactory>
+
+      <termination>
+        <secondsSpentLimit>10</secondsSpentLimit>
+      </termination>
+    </solver>
+  </inheritedSolverBenchmark>
+
+  <solverBenchmark>
+    <name>Default</name>
+    <!-- Intentionally empty for default solver -->
+    <solver>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Move Selector</name>
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <changeMoveSelector/>
+          <swapMoveSelector/>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
```

Comment:
Looking at it again, I think that these two configs do the same thing.

Reply:
Ok I double checked with @ge0ffrey , the first configuration has a localSearch construction heuristic by default, and the second configuration does not. Is this a concern?

Reply:
Ok given that construction heuristics should be run, there really is no point in the `Move Selector` configuration since it will be equivalent to `Default` if construction heuristics are added (it's also included in the Pillar Move Selector configs). It makes sense to remove this `Move Selector` config.